build01/configuration.nix

```diff
@@ -25,9 +25,6 @@
   networking.hostName = "build01";
   networking.hostId = "d2905767";
 
-  # Emulate armv7 until we have proper builders
-  boot.binfmt.emulatedSystems = [ "armv7l-linux" ];
```

Comment:
I think I want to bring riscv64-linux in though... arm 32-bit is probably a dead-end for NixOS anyway.